packages/icns/src/index.ts

```diff
@@ -0,0 +1,75 @@
+import { CosmWasmClient } from '@cosmjs/cosmwasm-stargate';
+import { useChain, useWallet } from '@cosmos-kit/react';
+import useSWR from 'swr';
```

Comment:
hey, what is `swr` needed for?

Reply:
seems like you have already figured out :)